### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR, Paperless Delivery

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#108556]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Everything looks great overall, which is why we focused in on specific content stuff!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** Make sure the labels for paperless delivery options are clear and explicit about what they do. The current "Yes" is a little confusing, especially since there's no question on the page that's being answered.
- [x] **Consider:** I think the challenge of making the option labels clear and explicit may be easier with radio buttons rather than checkboxes. This is definitely not a "must" but I'd encourage you to play with that option a little more and to listen for feedback in testing that may point towards radio buttons instead of checkboxes.
- [ ] **Consider:** It may be worth exploring including a summary somewhere on the page that describes the current state, eg. you're receiving X via mail + email, and you're receiving Y via email only. That may also be a way to clarify the interaction... or it may become too cumbersome as more documents are added to the paperless delivery page. Again, definitely not a "must," just something to explore!

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by raquelou** [comment]
We are updating the labels for paperless delivery and including the research question of "do Veterans find the labels for paperless delivery options clear and explicit".

We will also be testing radio buttons vs checkboxes for selection methods.


We have recorded, for future exploration, the comment: It may be worth exploring including a summary somewhere on the page that describes the current state, eg. you're receiving X via mail + email, and you're receiving Y via email only. That may also be a way to clarify the interaction... or it may become too cumbersome as more documents are added to the paperless delivery page. Again, definitely not a "must," just something to explore!
